spv3/loader/src/SPV3.csproj

```diff
@@ -41,6 +41,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
```

Comment:
This will set SPV3.exe to compile to 64-bit and it will prevent users on 32-bit hardware from using the installer/launcher.
We have a surprisingly significant amount of players still using 32-bit hardware.

Reply:
Alternatively, we could make two new applications opened by the "Check running processes" button.
They will be essentially identical aside from the architecture they target.
We already have code that checks for 64-bit-only paths (`"%ProgramFiles(x86)%", "%SYSDIR%/WOW6432", REG:WOW6432Node`). 
We can check if these paths exists (or return NULL, in the case of registry keys), then invoke either the x86 or x64 process-scanning executable.
Then the we return a process-end code for the installer to know if halo1.dll was preloaded by the MCC process.

While writing this, I also considered embedding the two "executables" into SPV3.exe as assemblies/libraries so we could continue distributing a single executable file. However, I am unsure how well that would work when the main module (spv3.exe) targets x86_x64 (ANY).

Reply:
Alternatively, we can remove the code that checks for the DLLs in favour of simpy checking if a process is running. This does indeed weaken the verification for a legitimate copy of CEA; however, we could find an alternative way of checking if CEA is installed.

Scanning the process' memory for a known value at a known offset that proves CEA is installed would be the way to go without compromising 32-bit support, as initially proposed.